assets/users/utils.js

```diff
@@ -27,7 +27,7 @@ export function getUserTypes(user) {
 }
 
 export function isUserAdmin(user) {
-    return get(user, 'user_type') === 'administrator';
+    return get(user, 'user_type') === 'Administrator';
```

Comment:
This is case sensitive, and is stored with the lowercase version `administrator`

Reply:
I checked the value of admin user type, it was `Administrator`. This helps in differentiating between admin and non admin user. And thus the PR would show delete option only for non admin user.